definitions/infra-awsvpclatticeservice/summary_metrics.yml

```diff
@@ -0,0 +1,17 @@
+AccountName:
+  tag:
+    key: aws.accountId
+  title: AWS account
+  unit: STRING
+byteSentRate:
+  goldenMetric: byteSentRate
+  unit: BYTES
+  title: Bytes sent per minute
+byteReceivedRate:
+  goldenMetric: byteReceivedRate
+  unit: BYTES
+  title: Packets received per minute
```

Comment:
Is the title correct? The unit and the metric names says bytes but the title PAckets

Reply:
You are correct! just updated it.